src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -428,6 +428,25 @@ sealed class RsDiagnostic(
         }
     }
 
+    class ExplicitCallToDrop(
+        element: PsiElement
+    ) : RsDiagnostic(element) {
```

Comment:
It's better to pass only identifier of method call/call expr not to highlight argument list because it can interfere with some other highlighting (for example, possible error annotation about incorrect parameters)